### ISSUE TITLE
Is this printer.cfg known good?

### ISSUE THREAD

**Issue by asagot** [DaCookie4u/klipper-cfg-troodon#1]
I just got a T300 and wanted to know if this is a known good printer.cfg file? Also, do you have a slicer profile that works well with it?

**Comment by DaCookie4u** [comment]
Yes, I have it running for a while now and it works just fine. I did removed the original display and switched to KlipperScreen though. You might want to tune some of the parameters and the macros.

You can find a mount for rpi and original screen here: https://www.printables.com/model/175176-troodon-raspi-mount
And an alternative for the 7" rpi touchdisplay here: https://www.printables.com/model/276590-troodon-7-display-mount

If you want to know how to connect the original display to your klipper check Robs website and his repository. The link are in my README.md.

**Comment by asagot** [comment]
Thanks - which slicer do you use?

**Comment by asagot** [comment]
Which parameters and macros need tuning?

**Comment by DaCookie4u** [comment]
I am using SuperSlicer and my profile is based on the PIF profile from @AndrewEllis93: https://github.com/AndrewEllis93/Ellis-PIF-Profile

Also checkout his Print Tuning Guide it is priceless: https://github.com/AndrewEllis93/Print-Tuning-Guide/

Regarding the parameters and macros which need tuning: I have no idea. Try the config and see whether everything suits you and change what doesn't suit you. Read the klipper documentation if this is your first time using klipper!

You will probably have to modify the endstop positions for x and y and the e-steps/rotation distance for the extruder. You will also have to pid tune the bed and hotend and adjust your bltouch z_offset. You will also want to check the PRINT_START and PRINT_END macros to configure your slicer accordingly.
